Packs/Carbon_Black_Enterprise_Response/Integrations/README.md

```diff
@@ -343,6 +343,9 @@ List the CarbonBlack sensors
 | Endpoint.Hostname | unknown | Sensor Hostname | 
 | Endpoint.OS | unknown | Sensor OS | 
 | Endpoint.IPAddresses | unknown | Sensor IP Addresses | 
+| Endpoint.ID | unknown | Sensor ID | 
+| Endpoint.Status | unknown | Sensor Status | 
+| Endpoint.IPAddress | unknown | Sensor IP Address | 
```

Comment:
I think they all need to be string. please tend to it in the yml as well.

Reply:
It's an old yml and the type is not mentioned in other context outputs so I tried to keep it the same.
Same about `CrowdStrike` readme (in the yml it's defined as strings)
I changed it in both `readme` files.

Reply:
You can change it in the yml. if it is indeed a String, it is not a BC break.